packages/maps/src/Client.ts

```diff
@@ -92,3 +92,56 @@ export interface MapOptions {
     geometryType: boolean
   };
 }
+
+export interface MapInstance {
+  layergroupid: string;
+  last_updated: string;
```

Comment:
last_updated ? vs updated_at --> Is the spec not updated? https://carto.com/developers/maps-api/reference/#operation/instantiateAnonymousMap

Reply:
`last_updated` is the one that I took from an actual request. Seems like that the spec not updated :(

Reply:
I created an issue at dev-center for them